cmd/topograph/main.go

```diff
@@ -27,15 +27,18 @@ import (
 	"k8s.io/klog/v2"
 
 	"github.com/NVIDIA/topograph/pkg/config"
+	"github.com/NVIDIA/topograph/pkg/models"
 	"github.com/NVIDIA/topograph/pkg/server"
 )
 
 var GitTag string
 
 func main() {
 	var c string
+	var m string
 	var version bool
 	flag.StringVar(&c, "c", "/etc/topograph/topograph-config.yaml", "config file")
+	flag.StringVar(&m, "m", "", "model file to use when using the test provider")
```

Comment:
Suggestion: instead of passing model as a command line argument to topograph, pass it as a parameter in topology request. That way we could run different models without restarting topograph